btcpayserver/values.yaml

```diff
@@ -0,0 +1,115 @@
+# Default values for btcpayserver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+postgresql:
+  image:
+    tag: 15.4.0-debian-11-r10
+  global:
+    postgresql:
+      auth:
+        postgresPassword: ""
+        password: ""
+        database: btcpay
+        username: btcpay
+
+image:
+  repository: btcpayserver/btcpayserver
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+persistence:
+  enabled: true
+  ## Set storageClassName if you have no default defined. For geth you need an fast (ssd) disk to keep up
+  #storageClassName: ""
+  size: 1Gi
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  {}
+  # fsGroup: 2000
+
+securityContext:
+  # runAsNonRoot: true
```

Comment:
Can't set this because:
```
Error: container has runAsNonRoot and image will run as root (pod: "btcpayserver-76f567767b-lv2g8_btcpayserver(e9051ad7-cbd8-4f82-991d-128c9dc8a8f0)", container: btcpayserver)
```